src/main/scala/com/high-performance-spark-examples/dataframe/LoadSave.scala

```diff
@@ -32,13 +25,13 @@ case class LoadSave(sqlContext: SQLContext) {
 
     // Create DataFrame explicitly with specified schema
     val schema = StructType(List(
```

Comment:
Can we switch this back to the other schema? The PandaPlace has changed to contain RawPandas.